### PR TITLE
[astro] added example with german mappings

### PR DESCRIPTION
**Pull request by mueller-ma**

### CODE REVIEW

File: addons/binding/org.openhab.binding.astro/README.md [hunk 1]
```diff
@@ -162,3 +162,100 @@ then
...
+        Text item=ASTRO_DAYTIME
+        Text item=ASTRO_ZODIAC
+        Text item=ASTRO_ZODIAC_START
+        Text item=ASTRO_ZODIAC_END
+        Text item=ASTRO_SEASON
+    }
+}
+```
+
+Transform season.map
```
Comment (itheiss) [inline addons/binding/org.openhab.binding.astro/README.md]: Hi,

Thanks for contributing and sharing! I would like to suggest the following. Add a caption 'Transformation definitons:' and below simply add 'season.map' and 'zodiac.map'

Comment (mueller-ma) [inline addons/binding/org.openhab.binding.astro/README.md]: @itheiss This way?

Comment (itheiss) [inline addons/binding/org.openhab.binding.astro/README.md]: Yep that's what i meant. But I think you have to remove the <br> after 'Transformation definitons:' or is this valid?

Comment (mueller-ma) [inline addons/binding/org.openhab.binding.astro/README.md]: @itheiss I'm not that familiar with markdown, but I have seen that html works and without that \<br\> there wouldn't be a new line

Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.astro/README.md]: @mueller-ma Simply replace the <br> by an empty line and you will have a line break.

**Comment by kaikreuzer** [comment]
@mueller-ma Thanks for this PR.
We usually have exactly ONE example in English for very binding. I am wondering, if the Astro binding really requires to be treated exceptionally? We could end up having example 3 in French, example 4 in Japanese, etc., so I am not sure if this is a good idea to start with in the first place.

What is the underlying issue that you want to address? Having German map files available? Maybe we find some other way, more general way to achieve this?

**Comment by mueller-ma** [comment]
@kaikreuzer First achievment: Copy and paste the config and you get the hole potential of the astro binding.

Second achievment: You can quickly download the mappings. Since they are here (https://github.com/openhab/openhab1-addons/wiki/Astro-binding) as well I thought it is a good idea to provide it to the oh2 binding

**Comment by ThomDietrich** [comment]
I'd second @kaikreuzer 's concern. The documentation is held in english, I wouldn't start adding german, french and spanish examples just for ease of use. that's what the "Tutorials&Examples" section in the community should be used for. There is by the way already a thread on this here, feel free to add your ideas there: 

https://community.openhab.org/t/astro-items-sitemap-binding-example/21155/1

**Comment by mueller-ma** [comment]
@ThomDietrich @kaikreuzer I see your concerns. It is actually bad to have different examples for each language. Have you thought about adding the link to the thread to the readme file?

**Comment by kaikreuzer** [comment]
> Have you thought about adding the link to the thread to the readme file?

This is also something I want to avoid on the readmes. Again, I think that is nothing specific to any binding. We should rather tell people that if they are looking for examples they should do a search in the tutorial&examples section of the forum (which imho has a brilliant search functionality).

**Comment by mueller-ma** [comment]
> We should rather tell people that if they are looking for examples they should do a search in the tutorial&examples section of the forum

I think it is good to link the tutorial&examples section in every readme file

**Comment by kaikreuzer** [comment]
Sounds fully redundant to me and it won't even be possible for ESH add-ons.

**Comment by ThomDietrich** [comment]
Your intention is to make it easier for users to find tutorials and examples to ease their start with openhab or specific bindings. Is that correct? Adding one link to every bindings readme is however redundant indeed. 

I guess what you are really looking for is a place in documentation pointing the user to the tutorials and examples forum category. There is already http://docs.openhab.org/tutorials and http://docs.openhab.org/appendix/help.html in place.
If you think these or other parts of the documentation page should be improved, please come to the docs Issue Tracker for further discussions: https://github.com/openhab/openhab-docs/issues

**Comment by marcelloceschia** [comment]
@mueller-ma the moon.map file is missing, so this example will cause `Caused by: java.io.FileNotFoundException: /etc/openhab2/transform/moon.map (Datei oder Verzeichnis nicht gefunden)`

**Comment by ThomDietrich** [comment]
@marcelloceschia a moon map is available in the [community forum example](https://community.openhab.org/t/astro-items-sitemap-binding-example/21155)

**Comment by mueller-ma** [comment]
@marcelloceschia The map file in the link above (https://raw.githubusercontent.com/ThomDietrich/openhab-config/master/transform/astro.map) is better than my missing file. Download it, save it as astro.map and replace all mappings in my example with astro.map

**Pull request closed without merging**